extensions/wikia/CreateNewWiki/tasks/CreateTables.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Wikia\CreateNewWiki\Tasks;
+
+class CreateTables implements Task {
+
+	use \Wikia\Logger\Loggable;
+
+	/** @var  TaskContext */
+	private $taskContext;
+
+	private $sqlFiles;
+
+	public function __construct($taskContext) {
+		$this->taskContext = $taskContext;
+	}
+
+	protected function getLoggerContext() {
+		return TaskHelper::getLoggerContext( $this->taskContext );
+	}
+
+	public function prepare() {
+		global $IP;
+
+		/* default tables */
+		$this->sqlFiles = array(
+			"{$IP}/maintenance/tables.sql",
+			"{$IP}/maintenance/interwiki.sql",
+			"{$IP}/maintenance/wikia/city_interwiki_links.sql",
+			"{$IP}/extensions/CheckUser/cu_changes.sql",
+			"{$IP}/extensions/CheckUser/cu_log.sql",
+			"{$IP}/maintenance/archives/wikia/patch-watchlist-improvements.sql",
+			"{$IP}/maintenance/archives/wikia/patch-create-blog_listing_relation.sql",
+			"{$IP}/maintenance/archives/wikia/patch-create-page_vote.sql",
+			"{$IP}/maintenance/archives/wikia/patch-create-page_visited.sql",
+			//article comments list use by wall/forum
+			"{$IP}/extensions/wikia/ArticleComments/patch-create-comments_index.sql",
+			//wall history table
+			"{$IP}/extensions/wikia/Wall/sql/wall_history_local.sql",
+			"{$IP}/extensions/wikia/VideoHandlers/sql/video_info.sql",
+			"{$IP}/maintenance/wikia/wikia_user_properties.sql",
+		);
+
+		$additionalTables = array(
+			"{$IP}/extensions/wikia/AjaxPoll/patch-create-poll_info.sql",
+			"{$IP}/extensions/wikia/AjaxPoll/patch-create-poll_vote.sql",
+			"{$IP}/extensions/wikia/ImageServing/sql/table.sql",
+		);
+
+		foreach ( $additionalTables as $file ) {
```

Comment:
Actually I don't know why we do it like this - these are static files so should be just added to the list above or removed if they're absent (non-readable) or am I missing something?